### PR TITLE
Project selection doc

### PR DESCRIPTION
**Pull request by marioba**

### CODE REVIEW

**Comment by marioba** [comment]
@m-kuhn yes, it's still true. I'm working on merge the pages.

File: en/project-management/project-selection.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    /Android/data/ch.opengis.qfield/files
+
+Warning!
+--------
+By storing data in this folder, you risk losing data.
+The application-specific directory (`/Android/data/ch.opengis.qfield/`) will be deleted by the system if you uninstall QField from your Android device. This is important if you keep the data in the External sd card.
+
+How to turn external SD card into internal storage on Android 6 and 7.
+--------
+We recommend turn external SD card into internal storage.
```
Comment (m-kuhn) [inline en/project-management/project-selection.rst]:
```suggestion
We recommend to turn external SD cards to internal storage.
```

File: en/project-management/project-selection.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
+Project selection
+=================
+
+QField opens QGIS projects in the ".qgs" format. Currently it is not possible to open ".qgz" files.
+
+OField uses a customized file selector to open the projects. On choosing "Open project" from the QField menu, the file selector will be opened showing the roots:
```
Comment (m-kuhn) [inline en/project-management/project-selection.rst]: OField?

Comment (m-kuhn) [inline en/project-management/project-selection.rst]: "showing the following folders:" ?

I think root as a term is not widely known (except for (arch) linux geniouses)

File: en/project-management/project-selection.rst [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+- The primary media/shared storage directory returned by Environment.getExternalStorageDirectory()
+- The application-specific directories on all shared/external storage devices where the application can place persistent files it owns if they are not already included in the primary storage directory.
+
+Inside these directories, only subdirectories and files with the ".qgs" extension are shown.
+
+QField requires the project file to be physically on the device, so it is not possible to use providers that work with remote files (e.g. Google Drive). On the other hand you can use apps like syncthing or nextcloud (or dropbox and probably many others) that allow you tho physically sync the file to your device.
```
Comment (m-kuhn) [inline en/project-management/project-selection.rst]:
```suggestion
QField requires the project file to be physically on the device, so it is not possible to use providers that work with remote files (e.g. Google Drive). On the other hand you can use apps like `syncthing <https://syncthing.net/>_` or `nextcloud <https://nextcloud.com/>`_ (or dropbox and probably many others) that allow you tho physically sync the file to your device.
```

**Comment by m-kuhn** [comment]
Very good, thanks a lot!

File: en/project-management/project-selection.rst (hunk 2 above)
Comment (marioba) [reply en/project-management/project-selection.rst]: Ops :)

Comment (marioba) [reply en/project-management/project-selection.rst]: :)

Comment (mbernasocchi) [reply en/project-management/project-selection.rst]: also root is understood as rooting

Comment (marioba) [reply en/project-management/project-selection.rst]: yes, already changed.

File: en/project-management/project-selection.rst [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+.. code-block:: none
+
+    /Android/data/ch.opengis.qfield/files
+
+Warning!
+--------
+By storing data in this folder, you risk losing data.
+The application-specific directory (`/Android/data/ch.opengis.qfield/`) will be deleted by the system if you uninstall QField from your Android device. This is important if you keep the data in the External sd card.
+
+How to turn external SD card into internal storage on Android 6 and 7.
```
Comment (mbernasocchi) [inline en/project-management/project-selection.rst]: is it only those 2 versions?

File: en/project-management/project-selection.rst [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+--------
+We recommend to turn external SD cards to internal storage.
+
+NOTE: Please be sure you start with a blank SD card. It needs to be blank because when the card is converted to "Internal" storage, it will be formatted (wiped) and encoded (making the cards data only readable by that device).
```
Comment (mbernasocchi) [inline en/project-management/project-selection.rst]: empty instead of blank

File: en/project-management/project-selection.rst [hunk 6]
```diff
@@ -0,0 +1,40 @@
+Project selection
+=================
+
+QField opens QGIS projects in the ".qgs" format. Currently it is not possible to open ".qgz" files.
+
+QField uses a customized file selector to open the projects. On choosing "Open project" from the QField menu, the file selector will be opened showing the following locations:
+
+- The primary media/shared storage directory returned by Environment.getExternalStorageDirectory()
```
Comment (mbernasocchi) [inline en/project-management/project-selection.rst]: maybe too technical

File: en/project-management/project-selection.rst (hunk 4 above)
Comment (marioba) [reply en/project-management/project-selection.rst]: no, seems to be 6 and above...

**Pull request merged**